keylime/ima/ima.py

```diff
@@ -479,6 +525,11 @@ def runtime_policy_db_contents(runtime_policy_name: str, runtime_policy: str, tp
         runtime_policy_db_format["ima_policy"] = runtime_policy
     runtime_policy_bytes = runtime_policy.encode()
     runtime_policy_dict = deserialize_runtime_policy(runtime_policy)
+
+    err_msg = validate_runtime_policy(runtime_policy_dict)
+    if err_msg:
+        raise ImaValidationError(message=f"{err_msg}", code=400)
```

Comment:
Is not weird to wrap `jsonschema.validate` into a function that catch the exception, to here revert it and raise one if there is a returned string?

Reply:
I also changed this now.